app/helpers.py

```diff
@@ -49,6 +49,9 @@ def decorator(f):
         def wrapper(*args, **kwargs):
             # take the input
             data = request.get_json()
+            print(data)
```

Comment:
Mhmmm, is this a lingering print statement?

Reply:
Nice catch! Thank you!